client/app/dedicatedCloud/user/rights/dedicatedCloud-user-rights.html

```diff
@@ -23,91 +23,34 @@ <h2 data-ng-bind-html="'dedicatedCloud_USER_rights_management_by_datacenter' | t
-                            </td>
-                            <td class="text-center"
-                                data-translate="{{ 'dedicatedCloud_USER_right_'+right.vmNetworkRole }}">
-                            </td>
-                            <td class="text-center"
-                                data-translate="{{ 'dedicatedCloud_USER_right_'+right.networkRole }}">
-                            </td>
-                            <td class="text-center">
-                                <button type="button"
-                                        class="btn btn-link"
-                                        title="{{ 'dedicatedCloud_USER_right_modify' | translate }}"
-                                        data-ng-click="setAction('user/rights/edit/dedicatedCloud-user-rights-edit', { rightId: right.rightId })">
-                                    <span class="oui-icon oui-icon-pen_line" aria-hidden="true"></span>
-                                </button>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-                <div data-pagination-server-side="rightTable"
-                     data-pagination-server-side-function="$ctrl.loadUserRight"
-                     data-pagination-server-side-paginated-stuff="$ctrl.rights"
-                     data-pagination-server-side-table-loading="$ctrl.loading.rights">
-                </div>
-            </div>
-        </div>
-    </div>
+        <oui-datagrid rows-loader="$ctrl.loadUserRights($config)">
+            <oui-column data-title=":: 'dedicatedCloud_datacenter_count' | translate"
+                        data-property="datacenterName"
+                        data-type="text">
+            </oui-column>
+            <oui-column data-title=":: 'dedicatedCloud_USER_access_vsphere' | translate">
+                <span data-translate="{{ 'dedicatedCloud_USER_right_' + $row.right }}"></span>
+            </oui-column>
+            <oui-column data-title=":: 'dedicatedCloud_USER_resource' | translate">
+                <span data-translate="common_yes"
+                      data-ng-if="$row.canAddResource">
+                </span>
+                <span data-translate="common_no"
+                      data-ng-if="!$row.canAddResource">
+                </span>
+            </oui-column>
+            <oui-column data-title=":: 'dedicatedCloud_USER_VM_NETWORK_ROLE' | translate">
+                <span data-translate="{{ 'dedicatedCloud_USER_right_' + $row.vmNetworkRole }}"></span>
+            </oui-column>
+            <oui-column data-title=":: 'dedicatedCloud_USER_NETWORK_ROLE' | translate">
+                <span data-translate="{{ 'dedicatedCloud_USER_right_' + $row.networkRole }}"></span>
+            </oui-column>
+            <oui-action-menu align="end" compact>
+                <oui-action-menu-item text="{{:: 'dedicatedCloud_USER_right_modify' | translate }}"
```

Comment:
Should use `data-text` / `data-on-click`

Reply:
Why isn't this detected by a linter :/